### PR TITLE
Simplify field writer unit test.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: searchlib/src/tests/diskindex/fieldwriter/fieldwriter_test.cpp [hunk 1]
```diff
@@ -273,6 +275,76 @@ WrappedFieldReader::close()
...
+}
+
+std::vector<vespalib::string> suffixes = {
+    "boolocc.bdat", "boolocc.idx",
+    "posocc.dat.compressed",
+    "dictionary.pdat", "dictionary.spdat", "dictionary.ssdat"
+};
+
+void
+check_fusion(const vespalib::string file_name_prefix)
```
Comment (geirst) [inline searchlib/src/tests/diskindex/fieldwriter/fieldwriter_test.cpp]: Did you forget &?

**Review by geirst — APPROVED** [review]
👍 Only a nitpick

File: searchlib/src/tests/diskindex/fieldwriter/fieldwriter_test.cpp (hunk 1 above)
Comment (toregge) [reply searchlib/src/tests/diskindex/fieldwriter/fieldwriter_test.cpp]: Fixed

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**